tests/includes/rest/class-test-outbox-controller.php

```diff
@@ -491,6 +502,183 @@ public function test_get_items_content_visibility( $visibility, $public_visible,
 		\wp_delete_user( $user_id );
 	}
 
+	/**
+	 * Test getting items with correct actor type filtering.
+	 *
+	 * @covers ::get_items
+	 */
+	public function test_get_items_actor_type_filtering() {
+		$user_id = self::factory()->user->create( array( 'role' => 'author' ) );
+
+		// Create a post with user actor type.
+		$user_post_id = self::factory()->post->create(
+			array(
+				'post_author'  => $user_id,
+				'post_type'    => Outbox::POST_TYPE,
+				'post_status'  => 'draft',
+				'post_title'   => 'https://example.org/activity/1',
+				'post_content' => wp_json_encode(
+					array(
+						'@context' => array( 'https://www.w3.org/ns/activitystreams' ),
+						'id'       => 'https://example.org/activity/1',
+						'type'     => 'Create',
+						'actor'    => 'https://example.org/user/' . $user_id,
+						'object'   => array(
+							'id'      => 'https://example.org/note/1',
+							'type'    => 'Note',
+							'content' => 'Test content',
+						),
+					)
+				),
+				'meta_input'   => array(
+					'_activitypub_activity_type'     => 'Create',
+					'_activitypub_activity_actor'    => 'user',
+					'activitypub_content_visibility' => ACTIVITYPUB_CONTENT_VISIBILITY_PUBLIC,
+				),
+			)
+		);
+
+		// Create a post with blog actor type.
+		$blog_post_id = self::factory()->post->create(
+			array(
+				'post_author'  => $user_id,
```

Comment:
Shouldn't this `0` then?

```suggestion
				'post_author'  => 0,
```